migrate/v0_15/incentive.go

```diff
@@ -86,30 +90,59 @@ func Incentive(incentiveGS v0_14incentive.GenesisState) v0_15incentive.GenesisSt
 	}
 
 	// Migrate Hard protocol claims (includes creating new Delegator claims)
+	hardClaims, delegatorClaims, err := migrateHardLiquidityProviderClaims(incentiveGS.HardLiquidityProviderClaims)
```

Comment:
For claims that only have delegator reward indexes, the migration ends up creating an empty `HardLiquidityProviderClaim`, that looks like:

```
{
        "base_claim": {
            "owner": "kava1qqw4ym0h3x6cp0lgscz2jp8kujnny98j0pzjst",
            "reward": []
        },
        "supply_reward_indexes": [],
        "borrow_reward_indexes": []
},
```

I assume there's no risk for future missing/incorrect rewards with having these in the store?

Reply:
good catch. If they don't have any borrow or deposits it shouldn't cause problems, but probably best to fix. It avoids unecessary state.

Reply:
Actually, not inlcluding these claims a bit complex. It's hard to drop them without first cleaning up the old state. Some claims with deposits/borrows are missing indexes.

Extra claims don't cause problems, when a new deposit/borrow/delegation is created any indexes in the claim are overwritten.

It would still be nice to clear out unecessary state. But I'll try do another PR, and prioritise merging this and the delegator PR.